sections/staking/components/StakingInfo/BurnInfo.tsx

```diff
@@ -29,32 +29,36 @@ const StakingInfo: FC = () => {
 		debtEscrowBalance,
 		collateral,
 	} = useStakingCalculations();
-	const synthsBalancesQuery = useSynthsBalancesQuery();
+
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { useSynthsBalancesQuery } = useSynthetixQueries();
+
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
 
 	const amountToBurn = useRecoilValue(amountToBurnState);
 
-	const sUSDBalance =
-		synthsBalancesQuery?.data?.balancesMap[Synths.sUSD]?.balance ?? toBigNumber(0);
+	const sUSDBalance = synthsBalancesQuery?.data?.balancesMap[Synths.sUSD]?.balance ?? wei(0);
 
 	const Rows = useMemo(() => {
-		const calculatedTargetBurn = Math.max(debtBalance.minus(issuableSynths).toNumber(), 0);
+		const calculatedTargetBurn = Math.max(debtBalance.sub(issuableSynths).toNumber(), 0);
 
-		const amountToBurnBN = toBigNumber(amountToBurn);
+		let amountToBurnBN = wei(0);
+
+		try {
+			amountToBurnBN = wei(amountToBurn);
+		} catch (_) {}
```

Comment:
we should do something in this catch

Reply:
yep just need to use the parseSafeWei function